keyboards/moonlander/moonlander.c

```diff
@@ -359,12 +359,12 @@ void keyboard_post_init_kb(void) {
 #if defined(AUDIO_ENABLE) && defined(MUSIC_MAP)
 // clang-format off
 const uint8_t music_map[MATRIX_ROWS][MATRIX_COLS] = LAYOUT_moonlander(
-    58, 59, 60, 61, 62, 63, 64,    65, 66, 67, 68, 69, 70, 71,
-    44, 45, 46, 47, 48, 49, 50,    51, 52, 53, 54, 55, 56, 57,
-    30, 31, 32, 33, 34, 35, 36,    37, 38, 39, 40, 41, 42, 43,
-    18, 19, 20, 21, 22, 23,            24, 25, 26, 27, 28, 29,
-     8,  9, 10, 11, 12,      3,     4,     13, 14, 15, 16, 17,
-                     0,  1,  2,     5,  6,  7
+    48, 49, 50, 51, 52, 53, 53,    54, 54, 55, 56, 57, 58, 59,
+    36, 37, 38, 39, 40, 41, 41,    42, 42, 43, 44, 45, 46, 47,
+    24, 25, 26, 27, 28, 29, 29,    30, 30, 31, 32, 33, 34, 35,
+    12, 13, 14, 15, 16, 17,            18, 19, 20, 21, 22, 23,
+     0,  1,  2,  3,  4,      5,     6,      7,  8,  9, 10, 11,
+                     5,  5,  5,     6,  6,  6
```

Comment:
These really should be different values.

Reply:
Why? Try it, it makes a lot of sense. Sure, the hit-area for 53, 54, 41, 42, 29, 30, 5, and 6 is rather large, but these are also the hardest to hit from the resting position on the home row.

Reply:
Hardest to hit is absolutely subjective.  Eg, on mine, hitting the different thumb keys is very easy for me, as my thumb sits in the center key in the thumb cluster, when my fingers are on the home row.  And I'm very used to using my thumb a lot.  So setting all of these keys to be the same thing is a literal waste.

And while I definitely understand that not everyone uses the board like me, ... well, that's still my point: that somebody likely will want to have these as different notes/positions.

What might be better/simpler in the long run is adding `__attribute__((weak))` to the variable, so that it can be changed in a users keymap.

Reply:
I've changed the PR so that it no longer picks a different default, but rather allows redefinition by the addition of `__attribute__((weak))`. Thanks for the suggestion, @drashna!